docs(accordion): updated accordion and accordion-item docs for accessibility (#5592)
adobe/spectrum-web-components: packages/accordion/accordion-item.md

## Overview

> The `<sp-accordion-item>` element represents a single item in an [`<sp-accordion>`](./accordion) parent element. Its `label` attribute and default slot content make up the "headline" and "body" of the toggleable content item.

Reviewer: ```suggestion
The `<sp-accordion-item>` element represents a single item in an [`<sp-accordion>`](./accordion.md) parent element. Its `label` attribute and default slot content make up the "headline" and "body" of the toggleable content item.
```

> The `<sp-accordion-item>` element represents a single item in an [`<sp-accordion>`](../accordion) parent element. Its `label` attribute and default slot content make up the "headline" and "body" of the toggleable content item.

Author: ```suggestion
The `<sp-accordion-item>` element represents a single item in an [`<sp-accordion>`](../accordion/) parent element. Its `label` attribute and default slot content make up the "headline" and "body" of the toggleable content item.
```

### Usage

[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/accordion?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/accordion)
[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/accordion?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/accordion)

> ```bash

Reviewer 2: Should this be `zsh`? It's the default for new macOS installs but the commands are compatible across both.

Author: ```suggestion
```zsh
```

```javascript
import { AccordionItem } from '@spectrum-web-components/accordion';
```

### Anatomy

The accordion item consists of two key parts:

- A header button with a label and chevron icon
- Content that is revealed when the item is expanded

```html
<sp-accordion>
    <sp-accordion-item label="Accordion Item">
        <div>Accordion item content can be toggled visible.</div>
    </sp-accordion-item>
</sp-accordion>
```

### States

#### Opened

An accordion item can be opened by default using the `open` attribute:

```html
<sp-accordion allow-multiple>
    <sp-accordion-item open label="Opened Accordion Item">
        <div>This accordion item content is visible by default.</div>
    </sp-accordion-item>
</sp-accordion>
```

#### Disabled

Individual accordion items can be disabled using the `disabled` attribute. Disabled items cannot be expanded or collapsed:

```html
<sp-accordion allow-multiple>
    <sp-accordion-item disabled label="Disabled Accordion Item">
        <div>You can not toggle this content visible.</div>
    </sp-accordion-item>
</sp-accordion>
```

### Behaviors

#### Events

> An `<sp-accordion-item>` element will dispatch `sp-accordion-item-toggle` events when it is opened or closed. By default, these events are dispatched to allow the parent `<sp-accordion>` to manage which of its item children can show their children at any one time. Consumers can also listen for this event and leverage `event.target.open` to ascertain the current state of the item dispatching the event.

Reviewer: ```suggestion
An `<sp-accordion-item>` element will dispatch `sp-accordion-item-toggle` events when it is opened or closed. By default, these events are dispatched to allow the parent `<sp-accordion>` to manage which of its item children to manage which of its item children are expanded at any given time.. Consumers can listen for this event and check event.target.open to determine whether the item is currently expanded.
```
